uaclient/tests/test_cli_attach.py

```diff
@@ -17,8 +17,12 @@
 
 BASIC_MACHINE_TOKEN = {
     "machineTokenInfo": {
-        "contractInfo": {"name": "mycontract", "resourceEntitlements": []},
-        "accountInfo": {"name": "accountName"},
+        "contractInfo": {
+            "id": "contract-1",
+            "name": "mycontract",
+            "resourceEntitlements": [],
+        },
+        "accountInfo": {"id": "acct-1", "name": "accountName"},
```

Comment:
It's not clear to me what's motivating this change being included in this PR. Am I missing some information?

Reply:
No motivation, I leaked it in from another branch accidentally. Removed and rechecked the full diff to confirm no stray commits from other branches.